apps/ui/src/components/ProposalsListItemHeading.vue

```diff
@@ -66,17 +66,18 @@ const space = computed(() =>
               space: `${proposal.network}:${proposal.space.id}`
             }
           }"
-          class="md:min-w-0"
+          class="md:flex"
         >
           <h3
-            class="text-[21px] inline [overflow-wrap:anywhere] md:truncate mr-2"
+            class="text-[21px] inline [overflow-wrap:anywhere] md:truncate mr-2 min-w-0"
```

Comment:
There are few other cases on home page and  http://localhost:8080/#/s:thanku.eth 🙈 

<img width="769" alt="Untitled 4" src="https://github.com/user-attachments/assets/529102c4-1337-4f02-942d-a8a51ff74e6d">